include/range/v3/view/ref.hpp

```diff
@@ -135,6 +135,13 @@ namespace ranges
         /// \ingroup group-views
         RANGES_INLINE_VARIABLE(view<ref_fn>, ref)
     }
+
+    namespace cpp20
+    {
+        CPP_template(typename Rng)(
+            requires std::is_object<Rng>::value)
+        using ref_view = ranges::ref_view<Rng>;
```

Comment:
Why not simply `using ranges::ref_view;`? A *using-declaration* here - instead of an *alias-declaration* - would preserve the capability to use CTAD.

Reply:
Good idea. I just followed what other headers did. The only difference seems to be the addition of requires clauses. Why not put those on the classes themselves and do as @CaseyCarter says, @ericniebler?

Reply:
Well, it's not like that'd do anything without concepts in the language. I'd guess that Eric chose having simulation of requires clauses over CTAD.

Reply:
Yes, CTAD and concept emulation are at odds. I haven't quite worked out what the general policy should be. Perhaps @johelegp is right and the `requires` clauses should be on the classes themselves. Needs thought. For now, it's probably best to be consistent with what the rest of the library does -- as you're doing here -- and I can fix it all at once.